### PR TITLE
feat(http api): update doc with new easy way to (de)activate

### PR DESCRIPTION
**Pull request by educhastenier**
HTTP API from version 7.13.0

### CODE REVIEW

**Review by vbotnari12 — COMMENTED** [review]
Ok for me.

File: modules/identity/pages/ldap-synchronizer.adoc [hunk 1]
```diff
@@ -711,19 +711,21 @@ Caused by: org.apache.http.client.HttpResponseException: status code: 404
...
+the Engine HTTP API by: 
+
+* either add a Java system property `-Dhttp.api=true` to file `setEnv.[sh|bat]` inside tomcat bundle (in folder `setup/tomcat_templates/`)
+* or set an environment variable `HTTP_API=true` before launching Bonita Tomcat bundle
+
+*Docker image solution:*
+
+Run the docker image with *-e HTTP_API=true*
+
+* For example: `docker run -e HTTP_API=true -e REST_API_DYN_AUTH_CHECKS=false --name bonita -v ./bonita-lic/:/opt/bonita_lic/ -h localhost -d -p 8080:8080 quay.io/bonitasoft/bonita-subscription:pass:a[{bonitaVersion}]`
```
Comment (baptistemesta) [inline modules/identity/pages/ldap-synchronizer.adoc:730]: That will fail because of the missing `HTTP_PASSWORD` (I'm not sure about the variable name)

**Review by baptistemesta — CHANGES_REQUESTED** [review]
All good. IMHO, the only thing to change is the command line to run docker with HTTP that require a password

File: modules/identity/pages/ldap-synchronizer.adoc (hunk 1 above)
Comment (educhastenier) [reply modules/identity/pages/ldap-synchronizer.adoc:730]: @baptistemesta no, on 2021.2, this line is valid.
And in 2022.1, it has been updated already.

nothing wrong here 😉

**Pull request merged**